ipyparallel/client/magics.py

```diff
@@ -29,6 +29,7 @@
 from __future__ import print_function
 
 import ast
+from contextlib import nullcontext
```

Comment:
Looks like `nullcontext` is not available on Py3.6 :(

Reply:
In its absence, we can define our own:

```python
@contextmanager
def nullcontext():
    yield
```

Reply:
Done. Thanks.